bi_sql_editor/models/bi_sql_view_field.py

```diff
@@ -108,7 +108,8 @@ def _compute_index_name(self):
                 sql_field.bi_sql_view_id.view_name, sql_field.name)
 
     # Overload Section
-    @api.multi
+    @api.model
+    @api.returns('self', lambda value:value.id)
```

Comment:
i don't understand why adding the second decorator. (@api.returns). Should works without dont you think ?

other wise, LGTM.

Reply:
@legalsylvain looking at how the create is used in the code, it seems the return value must be an id, not a recordset

Reply:
@legalsylvain just as @astirpe saids, that is how the code is decorated on main models.py odoo function

Reply:
Sorry, I don't understand. The function currently return a recordset.

see : ``return super(BiSQLViewField, self).create(vals)``

So, I think that @api.model will works.